src/accordion/accordion.spec.ts

```diff
@@ -26,6 +24,13 @@ function expectOpenPanels(nativeEl: HTMLElement, openPanelsDef: boolean[]) {
   expect(result).toEqual(openPanelsDef);
 }
 
+function createTestComponent(html: string, TestComponent: any): ComponentFixture<any> {
```

Comment:
This would not fit in `TestBed.overrideComponent` as it requires `concreteType<any>`